packages/moleculer-db-adapter-mongoose/src/index.js

```diff
@@ -67,17 +67,15 @@ class MongooseDbAdapter {
 		let conn;
 
 		if (this.model) {
-
 			/* istanbul ignore next */
 			if (mongoose.connection.readyState == 1) {
 				this.db = mongoose.connection;
 				return Promise.resolve();
 			} else if (mongoose.connection.readyState == 2) {
-				conn = Promise.resolve(mongoose.connection);
+				conn = mongoose.connection.$initialConnection;
```

Comment:
It looks it's an internal variable and I don't like that we use it.

@vkarpov15 could you help us? Can we use the `$initialConnection` or is there any better option to get the already connected connection?

Reply:
Thank you Valeri!
@thib3113 could you change the PR based on this comment, plz?

Reply:
@icebob ok . but what about the rest of the PR ? did we continue in this direction ( so, we don't handle two different url ) . Or we rework this part to handle differents url ?

Reply:
If we can't solve it, keep the current logic which doesn't support different urls.